frameworks/hdfs/tests/test_auth.py

```diff
@@ -71,6 +72,7 @@ def configure_package(configure_security):
         kerberos.cleanup()
 
 
+@pytest.mark.skipif(os.environ.get("SECURITY") == "strict", reason="auth tests currently broken in strict")
```

Comment:
There is an `sdk_utils.is_strict_mode()` function that can be used here instead.